### DISCUSSION TITLE
请问诚信承诺书可以在上面打字签名、学号和日期吗

### DISCUSSION THREAD

**Discussion by RoserX** [nju-lug/NJUThesis#259] [Q&A]
诚信承诺书的签名、学号和日期可以直接在上面打字吗

**Comment by atxy-blip** [comment]
学号和日期直接使用填入的个人信息，签名无法附图。这个承诺书建议使用 Word 版，填好、打印、签字、扫描后，使用 `includepdf` 插入。具体方法在手册中也注明了。
